index.js

```diff
@@ -3,25 +3,30 @@ const EventCalendar = require('./src/EventCalendar');
 
 $(function () {
     const calendar = new EventCalendar($('html'));
-    
-    /**
-     * Simple init() method which will default to startDate = today / endDate = today + 15 years
-     */
-    calendar.init();
-
-    /**
-     * More detailed example with saved events, start date and end date
-     */
 
-    // let myEvents = [
-    //     { date: '2019-01-25' },
-    //     { date: '2019-02-10' }
+    // let toAdd = [
+    //     '2019-02-05',
+    //     '2019-02-06',
+    //     '2019-02-09',
+    //     '2019-02-10',
+    //     '2019-02-11',
+    //     '2019-02-12'
     // ];
 
-    // let startDate   = '2019-01-10';
-    // let endDate     = '2019-03-20';
+    // let toDel = [
+    //     '2019-02-08',
+    //     '2019-02-15'
+    // ];
 
-    // calendar.init(startDate, endDate, myEvents);
+    /**
+     * Simple init() method which will default to startDate = today / endDate = today + 15 years
+     */
+    calendar.init({
+        startDateField: '.js-ercal-start',
+        endDateField: '.js-ercal-end'
+        // datesToAdd: toAdd,
+        // datesToDel: toDel
+    });
```

Comment:
I think we should delete any comments that aren't documenting, we can always get them back.